src/app/monitoring/containers/search-logs/search-logs.component.scss

```diff
@@ -1,3 +1,8 @@
+
+
+$text-color: gray;
```

Comment:
+ for using variables but are these the colors we are using globally in the project?